File: tests/comby/testdata/go/sample.go
package main

func oldFunc(args string) string {
	return args
}

func main() {
	oldFunc("test")
	oldFunc("another")

	func_call("error")
	func_call("success")
}

func swap(a, b int) (int, int) {
	return a, b
}

File: tests/comby/testdata/rs/sample.rs
fn main() {
    let x = Some(42);
    let y = Ok("test");
    
    dbg!(x);
    dbg!(y);
    
    println!("debug");
    println!("more debug");
    
    obj.old_field = 10;
    another.old_field = 20;
    
    let result = value.unwrap();
    let another = other.unwrap();
}

fn oldFunc(args: &str) -> &str {
    args
}
